src/components/input-switch/input-field.tsx

```diff
@@ -57,6 +57,19 @@ export type InputFieldProps = {
    * by default, we are capturing the "enter" key event and stopping it
    */
   allowEnter?: boolean
+  /**
+   * `optional`additional controller rules for the input field.
+   *  Check allowed rules here - https://react-hook-form.com/docs/useform/register#options
+   */
+   additionalControllerRules?: {
```

Comment:
I had used this in the previous version of arrayField to pass additional props for Validation.
 I am no longer using this, but I left the overall flow of passing additional props in case there is a need for this in future for any other component.
 
 In case you don't see a possible use case where this might be useful, let me know. I'll remove it.

Reply:
Yeah, let's leave it then.